packages/skin/src/routes/_index/component/card/+page.marko

```diff
@@ -0,0 +1,566 @@
+                </div>
+            </div>
+        </span>
+
+        <span class="card card--aspect-16-9">
+            <div class="card__media">
+                <img src=imgCards alt="Card Hero Image" class="card__hero-image">
+            </div>
+            <div class="card__main">
+                <div class="card__description">
+                    Hockey cards collected by enthusiasts who treat them like fine wine – stored in temperature-controlled cellars and discussed in hushed, reverent tones.
+                </div>
+                <div class="card__footer">
+                    <button class="btn btn--primary btn--fluid">See Details</button>
+                </div>
+            </div>
+        </span>
+    </div>
+</demo>
+
+<highlight-code type="html">
+<span class="card">
+    <div class="card__media">
+        <img src=imgCards alt="Card Hero Image" class="card__hero-image">
+    </div>
+    <div class="card__main">
+        <div class="card__description">
+            Wayne Gretzky's rookie card is so valuable that collectors keep it in climate-controlled vaults guarded by trained penguins.
+        </div>
+        <div class="card__footer">
+            <button class="btn btn--primary btn--fluid">See Details</button>
+        </div>
+    </div>
+</span>
+
+<span class="card card--aspect-5-4">
+    <div class="card__media">
+        <img src=imgCards alt="Card Hero Image" class="card__hero-image">
+    </div>
+    <div class="card__main">
+        <div class="card__description">
+            Football rookie cards so pristine that they've never been touched by human hands, only examined through specialized gloves by card whisperers.
+        </div>
+        <div class="card__footer">
+            <button class="btn btn--primary btn--fluid">See Details</button>
+        </div>
+    </div>
+</span>
+
+<span class="card card--aspect-16-9">
+    <div class="card__media">
+        <img src=imgCards alt="Card Hero Image" class="card__hero-image">
+    </div>
+    <div class="card__main">
+        <div class="card__description">
+            Hockey cards collected by enthusiasts who treat them like fine wine – stored in temperature-controlled cellars and discussed in hushed, reverent tones.
+        </div>
+        <div class="card__footer">
+            <button class="btn btn--primary btn--fluid">See Details</button>
+        </div>
+    </div>
+</span>
+</highlight-code>
+
+    <h2 id="card-loaded">Loaded Card with All the Options</h2>
+
+    <p>
+        These example demonstrate cards with all the available options, including different content elements - a header with a hero image, an overline for subtle categorization, with or without an icon, a title, a body content, and a footer with an action button.
+    </p>
+        
+    <p>
+        Here, you can also see the second style variant that uses an outline without a background. This can be implemented by using the <highlight>.card--outlined</highlight> modifier.
```

Comment:
I think I also prefer this second style moving to its own section. Side by side seems okay on desktop, but narrow screen I think its better to have the explanatory text right above the example.